src/Worker.cs

```diff
@@ -75,11 +76,16 @@ private static void WarmUpPowerShell(System.Management.Automation.PowerShell fir
                 .InvokeAndClearCommands();
         }
 
-        private static void LogPowerShellVersion(System.Management.Automation.PowerShell pwsh)
+        private static void LogPowerShellVersion(string pwshVersion)
         {
-            var message = string.Format(PowerShellWorkerStrings.PowerShellVersion, Utils.GetPowerShellVersion(pwsh));
+            var message = string.Format(PowerShellWorkerStrings.PowerShellVersion, pwshVersion);
             RpcLogger.WriteSystemLog(LogLevel.Information, message);
         }
+
+        private static string GetPowerShellVersion(System.Management.Automation.PowerShell pwsh)
+        {
+            return Utils.GetPowerShellVersion(pwsh);
```

Comment:
Could we use `Utils.GetPowershellVersion(pwsh)` instead of using this private method?

Reply:
Fixed.